GameOfLife/validator.h

```diff
@@ -0,0 +1,23 @@
+#ifndef VALIDATOR_H
+#define VALIDATOR_H
+#include <string>
+#include "sizewithframe.h"
+#include <fstream>
+#include <sstream>
+
+
+class Validator
+{
+public:
+    Validator();
+    bool validateIfStringHasAllLinesEqual(std::string);
+    bool validateIfStringHasInvalidCharacters(std::string);
```

Comment:
validateIfStringHasInvalidCharacters is now: validateIfStringHasNotInvalidCharacters